examples/example_manual_alignment.py

```diff
@@ -15,8 +15,8 @@
 
 
 def manual_alignment_example() -> None:
-    path_to_map = "./examples/localization-inspector.png"
-    path_to_echo_img = "./examples/Echo.png"
+    path_to_map = "./localization-inspector.png"
+    path_to_echo_img = "./Echo.png"
```

Comment:
At the moment, dependent on where the script is being run

```suggestion
    from os import path
    .
    .
    .
    here = path.dirname(__file__) + "/"
    path_to_map = here + "localization-inspector.png"
    path_to_echo_img = jere "Echo.png"
```